packages/@contentful--app-scripts/README.md

```diff
@@ -0,0 +1,77 @@
+# `@contentful/app-scripts`
+
+This project makes easy to perform some recurrent operations in [Contentful's App Framework](https://www.contentful.com/developers/docs/extensibility/app-framework/).
+
+## 📥 Installation
+
+Locally:
+
+```shell
+npm i --save @contentful/app-scripts
+```
+
+Globally:
+
+```shell
+npm i -g @contentful/app-scripts
+```
+
+
+## ⚙️ Usage
+
+### 💻 CLI
+
+When installed
+```
+$ contentful-app-scripts create-app-definition 
+```
+
+Otherwise
+```
+$ npx --no-install @contentful/app-scripts create-app-definition 
+```
+
+### 👨‍💻 Programmatic
+
+```javascript
+// my-script.js
+
+const { createAppDefinition } = require('@contentful/app-scripts')
+const { myCustomLogic } = require('./my-custom-logic')
+
+(async function main() {
+  myCustomLogic();
+  
+  createAppDefinition.interactive()
+})()
+```
+
+## 📜 API
+
+Scripts exported from this module will all be in the following shape
+
+```typescript
+interface Script<Result, Options> {
+  // query the user or local cache for required information
+  interactive: () => Result;
+  // run by automation (`--ci` flag), they would need all the information upfront
+  nonInteractive: (...options: Options) => Result;
+}
+```
```

Comment:
Do we really want to export an interactive API? I have never seen that before and also it feels wrong to me. As I user I never expect an API to show prompts to fill out missing options

Reply:
Maybe to plug it with your own flow: we're using standard tooling (`inquirer`, `commander`), it does not seem out of this world. 

OTOH, I am not super opinionated here and it felt like an easy win. Happy to tick it off

Reply:
Maybe it would be good to get more opinions on this?